task/dev9/src/com/github/zubarevladimir/Main.java

```diff
@@ -0,0 +1,41 @@
+package com.github.zubarevladimir;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+/**
+ * Contains entry-point method.
+ */
+public class Main {
+
+  /**
+   * Entry point metod.
+   *
+   * @param args contain file name.
+   */
+  public static void main(String[] args) {
+    try {
+      ReadFilesText readFilesText = new ReadFilesText();
+      String text = readFilesText.readFileToString(args[0]);
+      IniMapInitializer initializer = new IniMapInitializer();
+      ReadString reader = new ReadString();
+      System.out.print("Enter section: ");
+      String section = reader.readString();
+      System.out.print("Enter key: ");
+      String key = reader.readString();
+      Searcher searcher = new Searcher();
+      List<String> valueArray = searcher.searchValue(initializer.iniMap(text), section, key);
+      System.out.print("Value: \n");
+      for (String s : valueArray) {
+        System.out.println(s);
+      }
```

Comment:
no need to specialize exceptions catch, all of catches do the same thing- only logging. You can use simply Exception (in this version it is not processed)